wdn/templates_4.1/scripts/navigation.js

```diff
@@ -665,6 +665,10 @@ define(['jquery', 'wdn', 'modernizr', 'require'], function($, WDN, Modernizr, re
 						switchSiteNavigation($homepageCrumbLink[0], false);
 					});
 
+					$(navSel + ' a').last().focusout(function(){
```

Comment:
Gotcha. It will fail when the navigation switcher puts new navigation in the navigation container, thus dropping the old nav and the listener on the `<a>`. 

So you are saying add a listener to the parent `nav` container and listen to all `focusout` events, when the trigger is the last `<a>`, close the nav.

That makes sense, but it seems like a lot of overhead. I can't think of a better way of doing it.